### PR TITLE
Check for previously bagged file

### PR DESCRIPTION
**Pull request by tallenaz**
## Why was this change made?

Connects to https://github.com/sul-dlss/preservation_robots/issues/211

There are a subset of druids (including the fixture at https://github.com/sul-dlss/moab-versioning/commit/e718371fe99328b46885ffacbc9bcbb313c0714f) which contain duplicate entries in the content metadata that will cause the `Bagger` to try to bag the same file twice. This checks whether a file exists before bagging it up.

## Was the usage documentation (e.g. wiki, README) updated?

n/a

### CODE REVIEW

File: spec/fixtures/storage_root03/moab_storage_trunk/bb/077/xv/1376/bb077xv1376/v0001/data/metadata/technicalMetadata.xml [hunk 1]
```diff
@@ -0,0 +1,153 @@
+<technicalMetadata xmlns:jhove="http://hul.harvard.edu/ois/xml/ns/jhove" xmlns:mix="http://www.loc.gov/mix/v10" xmlns:textmd="info:lc/xmlns/textMD-v3" objectId="druid:bb077xv1376" datetime="2019-07-04T10:25:13Z">
```
Comment (jcoyne) [inline spec/fixtures/storage_root03/moab_storage_trunk/bb/077/xv/1376/bb077xv1376/v0001/data/metadata/technicalMetadata.xml:1]: we don't preserve technical metadata anymore, do we still need this file?

File: spec/fixtures/storage_root03/moab_storage_trunk/bb/077/xv/1376/bb077xv1376/v0001/data/metadata/workflows.xml [hunk 2]
```diff
@@ -0,0 +1,30 @@
+<workflows objectId="druid:bb077xv1376">
```
Comment (jcoyne) [inline spec/fixtures/storage_root03/moab_storage_trunk/bb/077/xv/1376/bb077xv1376/v0001/data/metadata/workflows.xml:1]: new objects don't have a workflows datastream either, can this be removed?

File: spec/fixtures/storage_root03/moab_storage_trunk/bb/077/xv/1376/bb077xv1376/v0001/data/metadata/technicalMetadata.xml (hunk 1 above)
Comment (tallenaz) [reply spec/fixtures/storage_root03/moab_storage_trunk/bb/077/xv/1376/bb077xv1376/v0001/data/metadata/technicalMetadata.xml]: Removed, and tests still pass

File: spec/fixtures/storage_root03/moab_storage_trunk/bb/077/xv/1376/bb077xv1376/v0001/data/metadata/workflows.xml (hunk 2 above)
Comment (tallenaz) [reply spec/fixtures/storage_root03/moab_storage_trunk/bb/077/xv/1376/bb077xv1376/v0001/data/metadata/workflows.xml]: Removed, and tests still pass.

**Comment by tallenaz** [comment]
I pared the fixture down to just the pdf in content. Happy to try to pare it down further if folks would like.

**Pull request merged**